packages/ddp-client/test/livedata_connection_tests.js

```diff
@@ -746,11 +772,11 @@ Tinytest.add('livedata stub - method call before connect', function(test) {
   });
 });
 
-Tinytest.addAsync('livedata stub - reconnect', async function(test, onComplete) {
+Tinytest.addAsync('livedata stub - reconnect111', async function(test, onComplete) {
```

Comment:
maybe typo ?

it could be just reconnect

Reply:
yes, typo